workshop/exercise-6/README.md

```diff
@@ -1,57 +1,73 @@
+          name: IMAGE
+        - default: /workspace/Dockerfile
+          description: Path to the Dockerfile to build.
+          name: DOCKERFILE
+        steps:
+        - args:
+          - --dockerfile=${DOCKERFILE}
+          - --destination=${IMAGE}
+          image: gcr.io/kaniko-project/executor
+          name: build-and-push
 	```
 
-	Output:
-	```
-		Routes in namespace 'default'
 
-		Name         Domain                                                                       Traffic                   Annotations  Conditions  Age  
-		fib-knative  fib-knative.default.bmv-knative-101-test.us-east.containers.appdomain.cloud  50% -> fib-knative-00002  -            3 OK / 3    9m  
-																								50% -> fib-knative-00001                             
+### Deploy the Fibonacci App Using kubectl and service.yaml
+
+1. Edit the `service.yaml` file to point to your own container registry namespace by replacing instances of <NAMESPACE> with the container registry namespace you created earlier.
 
-		1 routes
+2. Apply the `service.yaml` file to your cluster.
 
-		Succeeded
 	```
+	kubectl apply -f service.yaml
+	```
+3. Run `kubectl get pods --watch` to see the pods initializing.
 
-3. Let's run some load against the app, just asking for the first number in the Fibonacci sequence so that we can clearly see which revision is being called. Ensure you've replaced <ingress_subdomain> with your own ingress subdomain.
+4. Take a look at the `service.yaml` file again. The service.yaml file defines the required Knative components to build the application from source code in the git repository, push the built container image to the private container registry, and then run the application with the provide URL. 
+
+5. Now that the app is up, we should be able to call it using a number input. We can do that using a curl command against the URL provided to us. Esnure you've updated the command with your own ingress subdomain.
 
 	```
-	while sleep 0.5; do curl -X POST "http://fib-knative.default.<ingress_subdomain>/fib" -H 'Content-Type: application/json'   -d '{"number":1}' ; done
+	curl -X POST http://fib-knative.default.<ingress_subdomain>/fib -H 'Content-Type: application/json' -d '{"number":20}'
 	```
+6. You should see the first 20 Fibonacci numbers!
 
-4. We should see that the curl requests are routed approximately 50/50 between the two applications. Let's kill this process using ctrl-c.
+7. If we left this application alone for some time, it would scale itself back down to 0, and terminate the pods that were created. Run `kubectl get pods --watch` and wait until you see the application scale itself back down to 0. When the application is no longer in use, you should eventually see the pods move from the `Running` to the `Terminating` state.
 
+	Expected Output:
+	```
+	NAME                                            READY   STATUS      RESTARTS   AGE
+	fib-knative-00001-deployment-58dcbdb97c-rrnzc   3/3     Running     0          56s
```

Comment:
We should change the outputs shown to show 0002 - otherwise people might think something is wrong with their stuff.

Reply:
you may need to edit the follow-on exercises too since the revision numbers shown for knctl will be off as well

Reply:
okay, updated